### PR TITLE
DCOREUX-3862 update build tools

### PR DESCRIPTION
**Pull request by pts-michaelvera**
This project was using grunt and require.js, and I could not get it to build. So rather than fight it, I did a quick swap to webpack and NPM scripts.

### CODE REVIEW

File: documentation/app/index.html [hunk 1]
```diff
@@ -6,24 +6,26 @@
     <title>Points</title>
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-
-      <link rel="stylesheet" href="static/styles/app.css">
-      <link rel="icon" type="image/png" href="/static/img/favicon.png">
-
-
+    <link rel="stylesheet" href="/static/documentation.css">
```
Comment (pts-teddyhwang) [inline documentation/app/index.html]: Shouldn't this be a relative URL?

File: documentation/app/static/documents/getting-started.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ many loyalty programs. The LCP handles the complexity of working with each
 loyalty program, creating one simple interface for working with any number of
 programs.
 
-![Getting Started](static/images/getting-started.png)
+![Getting Started](../images/getting-started.png)
```
Comment (pts-teddyhwang) [inline documentation/app/static/documents/getting-started.md]: Do these relative URLs get updated in the build?

File: documentation/webpack.dev.js [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+var marked = require("marked");
+var path = require('path');
+
+var renderer = new marked.Renderer();
+
+const resolve = function resolve(target) {
+  return path.resolve(__dirname, target);
+};
+
+module.exports = {
```
Comment (pts-teddyhwang) [inline documentation/webpack.dev.js]: Lots of similarities with `webpack.config.js` - can we not extend and overwrite it?

File: documentation/app/static/documents/getting-started.md (hunk 2 above)
Comment (pts-michaelvera) [inline documentation/app/static/documents/getting-started.md]: Ya, webpack will update all the urls based on final paths of the files

File: documentation/webpack.dev.js (hunk 3 above)
Comment (pts-michaelvera) [inline documentation/webpack.dev.js]: Ya, got a little lazy, will address.

File: documentation/package.json [hunk 4]
```diff
@@ -30,10 +30,11 @@
...
     "url": "^0.11.0",
     "webpack": "^2.5.1",
-    "webpack-dev-server": "^2.4.5"
+    "webpack-dev-server": "^2.4.5",
+    "webpack-merge": "^4.1.0"
   },
   "scripts": {
     "start": "node scripts/start",
-    "build": "./node_modules/.bin/webpack"
+    "build": "./node_modules/.bin/webpack --config webpack.prod.js"
```
Comment (pts-teddyhwang) [inline documentation/package.json]: You can just use `webpack --config webpack.prod.js`. Npm is smart enough to look in the right `.bin` folder

**Pull request merged**